security-demo/cloud-stub/cmd/cloud_stub.go

```diff
@@ -19,6 +19,6 @@ func main() {
 
 	stub := cloudhub.NewCloudStub(config)
 
-	go stub.Start()
+	//go stub.Start()
```

Comment:
nit: please remove the dead line.

Reply:
Oops!! Accidental test code checked-in. Reverted back the changes.
Thanks for pointing out.